source/hotels.html

```diff
@@ -37,7 +37,55 @@
     </nav>
   </header>
   <main>
-    <!--форма "инфраструктура и тип жилья, стоимость"-->
+    <section>
+      <h2>Фильтр поиска</h2>
+      <form action="https://echo.htmlacademy.ru" method="get">
+        <img src="images/background-photo.jpg" width="1200" height="217" alt="">
+        <fieldset>
+          <legeng>Инфраструктура:</legeng>
+          <ul>
+            <li>
+              <input type="checkbox" name="swimingpool" id="filter-swimingpool" checked>
+              <label for="filter-swimingpool">бассейн</label>
```

Comment:
Когда инпут и лейбл лежат рядом, их можно писать вот так. Работает так же, как и через связку for + id
При клике на текст оно будет переводить курсор на поле. Переделай так все label + input
```suggestion
             <label>
               <input type="checkbox" name="swimingpool" checked>
               <span>бассейн</span>
             </label>
```